src/routes/search/containers/SearchModalContainer.js

```diff
@@ -39,8 +37,8 @@ const mapStateToProps = (state: StateType, ownProps: OwnPropsType) => {
     language: state.contentLanguage,
     cityCode: city,
     closeModal: () => { ownProps.navigation.goBack() },
-    sendFeedback: async (comment: string, query: string) => {
-      await feedbackEndpoint.request({
+    sendFeedback: async (comment: string, query: string, apiUrlOverride: ?string = undefined) => {
+      await createFeedbackEndpoint(apiUrlOverride || baseUrl).request({
```

Comment:
I think it would be cool if we'd have a utility function for this such that we never forget to use the override url

Reply:
Done.